docs/install/prepare-database-for-manager.rst

```diff
@@ -142,23 +95,22 @@ NOTE: All sub-commands under "schema" uses alembic.ini to establish database con
 Load initial fixtures
 ^^^^^^^^^^^^^^^^^^^^^
 
-Edit ``ai/backend/manager/models/fixtures.py`` so that you have a randomized admin keypair.
+The file ``fixtures/manager/example-keypairs.json`` , is example of admin keypair json file.
+If you want to change a randomized admin keypair, copy the ``fixtures/manager/example-keypairs.json`` to create and modify the ``example_keypair.json`` .
+so that you can edit & apply admin keypair using ``example_keypair.json`` file.
 
-:raw-html-m2r:`<span style="color:red">**(TODO: automate here!)**</span>`
+Database information is located on ``manager.toml`` file.
 
 Then pour it to the database:
 
 .. code-block:: console
 
-   $ python -m ai.backend.manager.cli \
-   >   --db-addr={DBHOST}:{DBPORT} \
-   >   --db-user={DBUSER} \
-   >   --db-password={DBPASS} 
-   >   --db-name=backend \
-   >   fixture populate example_keypair
+   $ ./backend.ai mgr fixture populate \
+   >   fixture populate example_keypair.json
```

Comment:
It seems that `fixture populate` is written twice here.

Reply:
Oops. I fixed that typos. Thanks @rapsealk